CoverMyDotNet/Requests/GetDrug.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using RestSharp;
+
+namespace CoverMyDotNet.Requests
+{
+	public class GetDrug : RestRequest
+	{
+		public GetDrug(string apiId, string id) : base()
```

Comment:
I wonder if it would be helpful to include the URL for the API documentation on each method of these client classes.  (e.g. for this one:  https://developers.covermymeds.com/ehr-api.html#get-forms)